src/NLog/Targets/EventLogTarget.cs

```diff
@@ -420,22 +401,18 @@ private void CreateEventSourceIfNeeded(string fixedSource, bool alwaysThrowError
                     {
                         // re-create the association between Log and Source
                         EventLog.DeleteEventSource(fixedSource, MachineName);
-                        var eventSourceCreationData = new EventSourceCreationData(fixedSource, Log)
-                        {
-                            MachineName = MachineName
-                        };
 
-                        EventLog.CreateEventSource(eventSourceCreationData);
+                        EventLog.CreateEventSource(new EventSourceCreationData(fixedSource, Log) { MachineName = MachineName });
```

Comment:
Why is this changed? I find it less readable now.

Reply:
Guess it's a matter of taste, I don't insist on this change. Same for ln:438.